simulator/mips/mips_instr.h

```diff
@@ -141,10 +144,9 @@ class FuncInstr
             static_assert( sizeof( AsR) == sizeof( uint32));
             static_assert( sizeof( AsI) == sizeof( uint32));
             static_assert( sizeof( AsJ) == sizeof( uint32));
-            static_assert( sizeof( uint32) == 4);
+            static_assert( sizeof( uint32) == 4);using Execute = void (FuncInstr::*)();
```

Comment:
Something strange.

Reply:
Realy strange. I see, that I delete your chages and try fast rework. Sorry

Reply:
Do not rush :-)